src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -68,6 +78,8 @@
 JOB_DIR_IN_ASSEMBLYUTIL_CONTAINER = "/kb/module/work/tmp"
 UPLOADED_YAML = "uploaded.yaml"
 
+AssemblyTuple = namedtuple("AssemblyTuple", ["assembly_name", "host_assembly_dir", "container_internal_assembly_path"])
```

Comment:
```suggestion
_AssemblyTuple = namedtuple("AssemblyTuple", ["assembly_name", "host_assembly_dir", "container_internal_assembly_path"])
```

Reply:
If I need to import this var in a test file, do I still need to make it `private`?

Reply:
Is this returned by a public method somewhere? Or are you bypassing the public interface for the test?

Reply:
like here: https://github.com/kbase/collections/blob/2e01b64bbf4adc3dc5fab6a298a1290d29f17114/test/src/loaders/workspace_uploader/workspace_uploader_test.py#L198

Reply:
So you're bypassing the public interface for that test since it's accessing a private function, so you might as well access private module variables as well

Reply:
👍

Reply:
:+1: